tox.ini

```diff
@@ -12,7 +12,7 @@ setenv = VIRTUAL_ENV={envdir}
          CHARM_LAYERS_DIR={toxinidir}/layers
          CHARM_INTERFACES_DIR={toxinidir}/interfaces
          JUJU_REPOSITORY={toxinidir}/build
-passenv = http_proxy https_proxy INTERFACE_PATH
+passenv = http_proxy,https_proxy,INTERFACE_PATH
```

Comment:
nit: most openstack repos put these on separate lines like setenv above, but if this is a backport we can leave it alone